tnc/data_handler.py

```diff
@@ -413,7 +432,7 @@ def send_data_to_socket_queue(self, /, **jsondata):
     def send_burst_ack_frame(self, snr) -> None:
         """Build and send ACK frame for burst DATA frame"""
         ack_frame = bytearray(14)
-        ack_frame[:1] = bytes([60])
+        ack_frame[:1] = bytes([FR_TYPE.ACK.value])
```

Comment:
This one should be named BURST_ACK so it's homogeneous to the other names.

Reply:
Done.